package.json

```diff
@@ -2,30 +2,30 @@
   "author": {
     "name": "Pierre Curto"
   },
-  "name": "lz4",
+  "name": "@scanreco/node-lz4",
   "description": "LZ4 streaming compression and decompression",
   "keywords": [
     "lz4",
     "compression",
     "decompression",
     "stream"
   ],
-  "version": "0.6.3",
-  "homepage": "http://github.com/pierrec/node-lz4",
+  "version": "0.6.5",
```

Comment:
Since this is not backward compatible, shouldn't we bump to a major version?

Reply:
0.7.0 or straight to 1.0.0?